.env.local-dev

```diff
@@ -8,3 +8,4 @@ EDC_UI_MANAGEMENT_API_KEY=no-api-key-required-in-local-dev
 EDC_UI_CATALOG_URLS=http://existing-other-connector/api/dsp,http://does-not-exist-but-is-super-long-so-we-can-test/api/dsp, http://how-wrapping-works-in-subtext-of-catalog-url-select/api/dsp
 EDC_UI_LOGOUT_URL=https://example.com/logout
 EDC_UI_CONNECTOR_ENDPOINT=http://localhost:3000/api/dsp
+EDC_UI_SHOWN_MANAGEMENT_API_URL_IN_DASHBOARD=http://localhost:3000/api/control/management
```

Comment:
I would like to revise my suggestion to `EDC_UI_MANAGEMENT_API_URL_SHOWN_IN_DASHBOARD`, please Ctrl+Shift+F twice